### ISSUE TITLE
FIFA Online 4 compatibility

### ISSUE THREAD

**Issue by Skoczek777** [DeathGOD7/FIFA-3D-Importer-Exporter#2]
FIFA 11 to FIFA 16 have their sections by which they name bone groups.
FO4 model is imported, but without bone groups and with messed up names.
It would be very big advantage.

**Comment by RexSE7EN** [comment]
Can you by chance show what happens if you import FO4 models? Like the blender screenshots or logs of the blender?

**Comment by Skoczek777** [comment]
Because it's not suited for FIFA Online 4, it names eye model as head and head model as eye and totally corrupts weights.
[2022-09-18.log](https://github.com/DeathGOD7/FIFA-3D-Importer-Exporter/files/9594410/2022-09-18.log)
![Zrzut ekranu (190)](https://user-images.githubusercontent.com/109217049/190917622-f01d2b56-9ea2-4165-8d8b-b805d7bc06a6.png)

**Comment by RexSE7EN** [comment]
> Because it's not suited for FIFA Online 4, it names eye model as head and head model as eye and totally corrupts weights. [2022-09-18.log](https://github.com/DeathGOD7/FIFA-3D-Importer-Exporter/files/9594410/2022-09-18.log) ![Zrzut ekranu (190)](https://user-images.githubusercontent.com/109217049/190917622-f01d2b56-9ea2-4165-8d8b-b805d7bc06a6.png)

Yeah the rx3 structure of FO4 is different from F14 so it might have been confused when applying weights.
